store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreTest.java

```diff
@@ -707,7 +709,8 @@ public void shouldNotAllowOverridingOfKnownGraphInLibrary() throws Exception {
     @Test
     public void shouldFederatedIfUserHasCorrectAuths() throws Exception {
         // Given
-        store.addGraphs(GRAPH_AUTHS_ALL_USERS, null, false, new GraphSerialisable.Builder()
+        final AccessPredicate noAccessPredicate = new NoAccessPredicate();
+        store.addGraphs(GRAPH_AUTHS_ALL_USERS, null, false, null, noAccessPredicate, new GraphSerialisable.Builder()
```

Comment:
Why is the addition of `NoAccessPredicate` required? Is that because the previous method in use was the one including disabledByDefault?

Reply:
@lb324567, think this was refactored by yourself. Any idea?

Reply:
I am not sure what happened here, as i don't remember making that change and it doesn't seem to be necessary. I'll undo this change.